examples/equilibria/coilset_opt_problem_tutorial.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
As discussed, this doesnt work as an ipynb. As agreed; leave this is as a script for now and drop the ipynb. Might need to add it to an ignore list.